modules/flake-parts/all-modules.nix

```diff
@@ -31,7 +31,13 @@ in {
   };
 
   # generates future flake outputs: `modules.<kind>.<module-name>`
-  config.flake.modules = lib.mapAttrs (kind: _: mapModules kind) moduleKinds;
+  config.flake.modules =
+    let modules = lib.mapAttrs (kind: _: mapModules kind) moduleKinds;
+    in modules // {
+      flake-parts = modules.flake-parts // {
+        all-modules = { imports = flakePartsModules; _class = "flake-parts"; };
```

Comment:
Kinds are actually called `class` in `evalModules` or `_class` in the module syntax.
It's a fairly new thing, not too long before 23.05. https://nixos.org/manual/nixpkgs/stable/#module-system-lib-evalModules-param-class

Haven't used `class` much yet as it's incompatible with 22.11, but that's increasingly no excuse anymore ;)